src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -280,10 +285,8 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
                 try
                 {
                     // open the input file
-                    using (FileStream fs = new FileStream(InFile, FileMode.Open))
-                    {
-                        SetRequestContent(request, fs);
-                    }
+                    _fileStream = new FileStream(InFile, FileMode.Open);
```

Comment:
This assignment is not needed. It can just be `SetRequestContent(request, new FileStream(InFile, FileMode.Open));`